File: event/src/test/java/de/danoeh/antennapod/event/MarkerEventsTest.java
package de.danoeh.antennapod.event;

import org.junit.Test;

import static org.junit.Assert.assertNotNull;

public class MarkerEventsTest {

    @Test
    public void streamingConfirmationEventConstructs() {
        assertNotNull(new StreamingConfirmationEvent());
    }

    @Test
    public void playerStatusEventConstructs() {
        assertNotNull(new PlayerStatusEvent());
    }

    @Test
    public void discoveryDefaultUpdateEventConstructs() {
        assertNotNull(new DiscoveryDefaultUpdateEvent());
    }

    @Test
    public void statisticsEventConstructs() {
        assertNotNull(new StatisticsEvent());
    }
}

File: event/src/test/java/de/danoeh/antennapod/event/MarkerEventsTest.kt
package de.danoeh.antennapod.event

import org.junit.Assert.assertNotNull
import org.junit.Test

class MarkerEventsTest {

    @Test
    fun streamingConfirmationEventConstructs() {
        assertNotNull(StreamingConfirmationEvent())
    }

    @Test
    fun playerStatusEventConstructs() {
        assertNotNull(PlayerStatusEvent())
    }

    @Test
    fun discoveryDefaultUpdateEventConstructs() {
        assertNotNull(DiscoveryDefaultUpdateEvent())
    }

    @Test
    fun statisticsEventConstructs() {
        assertNotNull(StatisticsEvent())
    }
}
